src/Shared/Traits.cs

```diff
@@ -179,6 +179,11 @@ public bool LogProjectImports
         /// </summary>
         public readonly bool DisableNuGetSdkResolver = Environment.GetEnvironmentVariable("MSBUILDDISABLENUGETSDKRESOLVER") == "1";
 
+        /// <summary>
+        /// Disable AssemblyLoadContext isolation for plugins.
+        /// </summary>
+        public readonly bool UseSingleLoadContext = Environment.GetEnvironmentVariable("MSBUILDSINGLELOADCONTEXT") == "1";
```

Comment:
Do you think this should be opt-in?

Reply:
No, we want the new behavior by default, because we expect it to be better for most people. This PR just buys us time to fix problems if it's not . . .